active_learning_multiannotator/active_learning.ipynb

```diff
@@ -0,0 +1,419 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using Cleanlab to Determine Examples to Collect Additional Labels for in an Active Learning Loop to Improve Classification Models "
```

Comment:
We want to be very clear to distinguish this from the traditional academic Active Learning. What is different in our setting is there are multiple annotators, and we can have annotators review already labeled examples in order to collect a higher quality overall set of labels.

Reply:
I edited the title slightly, but I'm not super certain on how to make it super clear and concise - any suggestions would be helpful!